### ISSUE TITLE
test TestTrack_FetchLyrics fails

### ISSUE THREAD

**Issue by quasilyte** [mamal72/golyrics#1]
```
go test -v -run=FetchLyrics github.com/mamal72/golyrics

=== RUN   TestTrack_FetchLyrics
=== RUN   TestTrack_FetchLyrics/should_fetch_lyrics_for_the_track_and_sets_it_on_it
--- FAIL: TestTrack_FetchLyrics (0.22s)
    --- FAIL: TestTrack_FetchLyrics/should_fetch_lyrics_for_the_track_and_sets_it_on_it (0.22s)
        golyrics_test.go:265: "should fetch lyrics for the track and sets it on it". Track.FetchLyrics() = The shortest song in the universe
            Really isn't much fun
            It only has one puny verse
            ... And then it's done!
            , want The shortest song in the universe
            Really isn't much fun
            It only has one puny verse
            . . . and then it's done!
FAIL
FAIL	github.com/mamal72/golyrics	0.228s
```

**Comment by quasilyte** [comment]
Should be fixed in #2.

**Comment by mamal72** [comment]
Just merged #2. TY. 🙏
